netbox/tenancy/forms/bulk_edit.py

```diff
@@ -90,8 +90,8 @@ class ContactRoleBulkEditForm(NetBoxModelBulkEditForm):
 
 
 class ContactBulkEditForm(NetBoxModelBulkEditForm):
-    group = DynamicModelChoiceField(
-        label=_('Group'),
+    groups = DynamicModelMultipleChoiceField(
```

Comment:
Because this is a many-to-many field, setting `groups` in bulk will wipe out any existing assignments. It would be ideal to include separate add & remove fields, though admittedly we don't have a great pattern for this today (aside from the assignment of tags).